api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -25,6 +41,37 @@ trait HttpApi extends HmdaApiProtocol {
           ToResponseMarshallable(status)
         }
       }
-    }
+    } ~
+      path("upload") {
+        post {
+          val processingActor = system.actorOf(ProcessingActor.props)
+          entity(as[Multipart.FormData]) { formData =>
+            val uploaded: Future[Done] = formData.parts.mapAsync(1) {
+              //TODO: check Content-Type type as well?
+              case b: BodyPart if b.filename.exists(_.endsWith(".csv")) =>
+                b.entity.dataBytes
+                  .via(splitLines)
+                  .map(_.utf8String)
+                  .runForeach(line => processingActor ! line)
+
+              case _ => Future.failed(throw new Exception("File is not CSV"))
```

Comment:
Are there other failure scenarios we need to consider besides just the wrong file extension?